GetMLBAthleteData.py

```diff
@@ -39,7 +39,7 @@ def computePrice(athlete_data, lgweightedOnBase, sumPlateAppearances):
 
         # Formula 
         avg50yrRPW = 9.757
-        collateralizationMultiplier = 1000
+        # collateralizationMultiplier = 1000
```

Comment:
Why comment instead of actually remove? Are you planning to use this later? If you are leave a note why it's commented and create a future ticket to keep the work to reintroduce it in scope make sure that ticket is mentioned in your comment. This prevents later devs from just removing this if it was actually needed later.

Reply:
noted & updated

Reply:
Since commented code often because dead code that no one remembers the use for and there are no actionable plans to uncomment I have found it best to create a specific ticket that would address it so it can be assessed during planning. 
Up to you though.